doc: specify Docker image steps (#379)
CleverCloud/documentation: content/doc/applications/docker/_index.md

[FS Buckets](/doc/best-practices/cloud-storage/#what-is-fs-bucket) access, Dockerfile validation, and Docker Compose functionalities are not supported.
{{< /callout >}}

### How it works

When you create a Docker application on Clever Cloud, the deployment process involves the following steps:

1. **Install/Login:**
   - The system checks for a Dockerfile and an entrypoint.

>    - It logs into the appropriate registry if necessary.

Reviewer: ```suggestion
   - It logs into a docker registry if configured
```

Author: I hesitated with this one because tagging Docker directly could be misleading. For example, Docker registry in GitHub is called Container registry.

Reviewer 2: May I suggest being extra verbose?

> It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (note: the name of the docker registry may vary depending on the provider. It is called "container registry" in Github, for instance)

(note: I'm not sure the docker registry is configure *inside the Dockerfile*)

Author: After internal discussion involving semantic king @Keksoj, this seems to be the best description:  
```suggestion
   -  It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (**note:** the name of the docker registry may vary depending on the provider. It is called "container registry" in Github, for instance)
```

2. **Build:**

>    - The application pulls the specified image an execute commands you specified in Dockerfile.

Reviewer: ```suggestion
   - The application pulls the specified image and execute commands you specified in Dockerfile.
```

Author: Thanks!

> - It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (**note:** the name of the docker registry may vary depending on the provider. It is called "container registry" in Github, for instance)

Author: ```suggestion
  - It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (**note:** the name of the docker registry may vary depending on the provider. It is called "container registry" in Github, for instance)
```

>   - It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (**note:** the name of the docker registry may vary depending on the provider. It is called "container registry" in Github, for instance)

Author: ```suggestion
    - It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (**note:** the name of the Docker registry may vary depending on the provider. It is called "container registry" in GitHub, for instance)
```

>     - It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (**note:** the name of the Docker registry may vary depending on the provider. It is called "container registry" in GitHub, for instance)

Author: ```suggestion
    - It logs into the docker registry that you configured in the Dockerfile, if any, to find the necessary image (**note:** the name of the Docker registry may vary depending on the provider. It's called "container registry" in GitHub, for instance)
```
